.github/workflows/main.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04]
-        pyodide-version: [0.22.1, 0.23.2]
+        pyodide-version: [0.24.0]
```

Comment:
Should we document a compatibility matrix somewhere? Of compatible Pyodide version for a given micropip release?